### PR TITLE
Updated README.md with STI_LOCATION variable description.

### PR DESCRIPTION
**Pull request by soltysh**
@jhadvig I did small enhancements in the readme :smiley:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,27 +1,27 @@
...
+The versions currently supported are:
 * **nodejs-0.10**
 
 
 Installation
 ---------------
-To build NodeJS image, choose between CentOS7 or RHEL7 base image:
-*  **RHEL7 base image**
+To build NodeJS image, choose between CentOS7 or RHEL7 based image:
+*  **RHEL7 based image**
```
Comment (mfojtik) [inline README.md]: I would strip the versions here, say just RHEL and Centos

**Comment by mfojtik** [comment]
[test]

**Comment by mfojtik** [comment]
[test]

File: README.md (hunk 1 above)
Comment (jhadvig) [inline README.md]: +1 on that

Comment (jhadvig) [inline README.md]: Scratch that ... I guess that it makes kinda sense to keep the OS version there (for now)

**Comment by jhadvig** [comment]
LGTM

**Comment by jhadvig** [comment]
[test]

File: README.md (hunk 1 above)
Comment (soltysh) [inline README.md]: I removed all the versions and added info at the beginning of the file which RHEL and CentOS versions we support.

**Pull request merged**